src/perf-tests/array.ts

```diff
@@ -0,0 +1,39 @@
+/**
```

Comment:
Thanks! Could you please wrap it in a Benchmark.Suite like in src\perf-tests\sort.ts and others.

Reply:
Would be curious to see worst case for sortIfNeeded where only the last two items are unsorted.